### ISSUE TITLE
Does (or can) pubsub .watch() support a selection filter?

### ISSUE THREAD

**Issue by shanemileham** [balderdashy/sails-docs#558]
The [.watch()](https://github.com/balderdashy/sails-docs/blob/master/reference/websockets/resourceful-pubsub/watch.md) function seems to watch ALL new models, while the [.subscribe()](https://github.com/balderdashy/sails-docs/blob/master/reference/websockets/resourceful-pubsub/subscribe.md) function gets changes to any existing model records in the database.

It would be very helpful to have a selective watch() function (just adding a query as an optional second arg) - can this be done currently, or is there a plan to add it? My reasoning for wanting this feature is explained [here](http://stackoverflow.com/questions/32414896/).

Thank you for your help.

**Comment by shanemileham** [comment]
And regarding the documentation, clarification that .subscribe() only gets existing models would be helpful (since its connotation led me to initially believe it would subscribe me to future model instances).

If I am misunderstanding either of these functions, please let me know. Thanks!

**Comment by sailsbot** [comment]
Thanks for posting, @shanemileham.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
